packages/datadog-metrics-tracker/src/base.ts

```diff
@@ -63,7 +63,7 @@ export abstract class MetricsTracker {
   private getInvalidTagNames(tags: { [name: string]: string }, measurementName: string) {
     try {
       return Object.entries(tags)
-        .filter(([_, value]) => value.length === 0)
+        .filter(([_, value]) => value?.length === 0)
```

Comment:
Should we have a test to cover this?

Reply:
yup and I'm working on it now, just getting help from Mani in setting up the environment for testing. I was thinking that it's critical and small enough to push the bugfix first and then create the new tests to cover it. What do you think?